source/release-notes/backend.txt

```diff
@@ -13,6 +13,15 @@ Atlas App Services Changelog
 2024 Releases
 -------------
 
+.. _backend_20240821:
+
+21 August 2024 Release
+~~~~~~~~~~~~~~~~~~~~~~
+
+- GraphQL and Static Hosting deprecation has begun. Apps currently with these 
+  features will continue to have access until March 2025. New Apps cannot enable
+  GraphQL or Static Hosting.
```

Comment:
This is just a nitpick, but we started the deprecation process when we announced the deprecation and started planning for it. I think we should remove the first sentence and simplify this messaging.
```suggestion
- New Apps cannot enable GraphQL or Static Hosting. Apps that already have 
  these features will continue to have access until March 2025.
```

Reply:
Perfect, thanks!